application/dash_application/pages.py

```diff
@@ -29,6 +34,17 @@
 
 # Please do not edit this loop
 for name in pagenames:
-    pages.append(importlib.import_module(prefix + name))
+    try:
+        pages.append(importlib.import_module(prefix + name))
+    except (IOError, OSError):
+        print(traceback.format_exc(), file=sys.stderr)
+        ErrorPage = namedtuple('ErrorPage', 'layout title dataversion page_name init_callbacks')
+        pages.append(ErrorPage(
+            lambda x: html.H3("Unexpected Error: Failed to load cache"),
```

Comment:
When this bug appears in the wild, it'll take a developer to fix it. Can we either add Prometheus monitoring that fires an alert for this case, or add a button for the user to submit a ticket describing the issue?

Reply:
Echoing this

Reply:
Error view contains a link that will generate a JIRA ticket.